packages/tenderly-core/src/internal/cli/commands/LoginCommand.ts

```diff
@@ -53,14 +63,20 @@ const validator = async function (value: string) {
 
 async function canAuthenticate(accessToken: string): Promise<boolean> {
   try {
+    logger.debug("Determining if the user can be authenticated with provided access token:", accessToken);
+
+    logger.trace("Making a call to the url...");
```

Comment:
Maybe a better message will be something like `checking if the user access token is valid...`